content/src/main/content/jcr_root/apps/pokegolure/components/manage/clientlibs/js/script.js

```diff
@@ -40,6 +43,10 @@ PokeGoLure.Manage = (function ($) {
         $signOutBtn.on('click', _handleSignOut);
         $luresList.on('click', '.pokego-manage__lures__item__delete', _handleLureDeleteClick);
         $luresList.on('mouseenter mouseleave', '.pokego-manage__lures__item', _handleLureMouseMovement);
+
+        _fetchPokeData();
+
+        setInterval(_fetchPokeData, 1000 * 60); // Fetch data every 60 seconds
```

Comment:
Should be fine, all it's doing is reading the player info and counting how many lures are in the player's inventory